Hunting Queries/MultipleDataSources/DisabledAccIPSigninWithRareRiskyOps.yaml

```diff
@@ -0,0 +1,91 @@
+id: 9adbd1c3-a4be-44ef-ac2f-503fd25692ee
+name: Successful Signin from IP with Disabled Account - Rare and potentially high-risk Office operations
```

Comment:
You might want to make this a bit more descriptive.

Perhaps something like "High risk Office operation conducted by IP Address that recently attempted to log into a disabled account"